src/data/customer/Setting.js

```diff
@@ -68,9 +71,35 @@ export const getCustomerSettingsAsync = async (token) => {
 
     return camelize(CustomerSettings.data);
   } catch (e) {
-    return Promise.reject({
-      statusCode: e.response.status,
-      statusText: e.response.statusText,
+    return apiResponseErrorHandler(e);
+  }
+};
+
+/** Retrieve Customer's Settings (reduced)
+ * @param {string} token
+ * @param {int} customerId
+ * @param {Object} payload {identityId, transactionGroupId}
+ * @return {Promise} settings object
+ */
+export const showCustomerSettingsAsync = async (
+  token,
+  customerId,
```

Comment:
I don't think customerId is required, as per documentation?

Reply:
@maartenhemmes we discussed it with Vinay and he sent me:

Reply:
@maartenhemmes API url is settings/{id}

id needs to be replaced by customerId

Reply:
@maartenhemmes it's done, I've replace id with customerId. Is it right? Or You mean that syntax should be id, but by logic it should be customerId?

Reply:
@vdanylov it's fine, if you want to optimise you could do the same as on other functions:

- make the {0} part of the url on Endpoint.js:   CUSTOMER_DRIVERS: `${urlPrefix}/customers/{0}/drivers`,

- Add a replace function:       url: endpoints.CUSTOMER_DRIVERS.replace('{0}', customerId),

This way it's consistent with the other functions.

Reply:
@maartenhemmes version changed to 0.1.99